docs/releases/4.2.txt

```diff
@@ -48,6 +48,9 @@ Minor features
   :attr:`~django.contrib.admin.ModelAdmin.filter_vertical` widgets are now
   filterable.
 
+* The ``autofocus`` attribute was removed from the search box due to accessibility
+   considerations.
```

Comment:
Maybe :thinking: :
```suggestion
* The ``autofocus`` HTML attribute in the admin search box is removed as it can
  be confusing for screen readers.
```